ietf-optical-impairment-topology.tree

```diff
@@ -249,11 +249,11 @@ module: ietf-optical-impairment-topology
           |     +--ro roadm-cd?
           |     |       l0-types:decimal-5-digits-or-null
           |     +--ro roadm-pdl?
-          |     |       l0-types:ratio-in-db-or-null
+          |     |       l0-types:loss-in-db-or-null
           |     +--ro roadm-inband-crosstalk?
-          |     |       l0-types:ratio-in-db-or-null
+          |     |       l0-types:loss-in-db-or-null
```

Comment:
no, inband crosstalk is not a loss, I prefer to keep ratio-in-db-or-null

Reply:
We have removed the ratio-in-db-or-null data type from l0-types: maybe better to defined it as a decimal-2-digits-or-null type with units "dB"

Reply:
hello, crosstalk is usually a positive value . if you remove the ratio-in-db, then definig decimal-2-digits is ok for me. 
I think that using loss in the type for this one is too far from its physical meaning.

Reply:
Addressed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/135/commits/62ec3365bb97f7d5b1a2b3135a70091c2f52cbc5

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>